pkgs/love2d/example/assets/love_example_browser/list.lua

```
@@ -0,0 +1,217 @@
+	if b == 1 and self:hasBar() then
+		local rx, ry, rw, rh = self:getBarRect()
+		if inside(x, y, rx, ry, rw, rh) then
+			self.bar.lock = { x = x, y = y }
+			return
+		end
+	end
+
+	if type(self.onclick) ~= "function" then return end
+
+	if inside(x, y, self.x + 2, self.y + 1, self.width - 3, self.height - 3) then
+		local tx, ty = x - self.x, y + self:getOffset() - self.y
```

Luacheck (1.2.0)

[warning] unused variable 'tx'

(W211)